testcontainers/src/core/image/image_ext.rs

```diff
@@ -169,6 +169,12 @@ pub trait ImageExt<I: Image> {
 
     /// Sets the user that commands are run as inside the container.
     fn with_user(self, user: impl Into<String>) -> ContainerRequest<I>;
+
+    /// Sets the container's root filesystem to be mounted as read-only
+    fn with_readonly_rootfs(self, readonly_rootfs: Option<bool>) -> ContainerRequest<I>;
+
+    /// Sets security options for the container
+    fn with_security_opt(self, security_opt: Option<Vec<String>>) -> ContainerRequest<I>;
```

Comment:
I think we can drop `Option` from arguments similarly to other methods.

I'd say `None` is needed as a default option, but isn't really needed as part of API here

Reply:
Fixed